src/docs/event.yaml

```diff
@@ -188,6 +189,50 @@ paths:
                 500:
                     description: Internal server error
     /api/event/:id:
+        get:
+          tags: 
+          - "Events"
+          summary: "Get event by id"
+          description: "Returns a single event"
+          operationId: "getEventById"
+          produces:
+          - "application/json"
+          parameters:
+            - name: "id"
+              in: "path"
+              description: "ID of the event to return"
+              required: true
+              type: "string"
+              format: "int64"
+          responses:
+            "200":
+              description: "An event by spesific id"
+              content: 
+               application/json:
+                schema:
+                type: "object"
+                items: 
+                  $ref: "#/components/schemas/Event"
+            "404":
+              description: Error Message
+              content:
+               application/json:
+                schema:
+                  type: "object"
+                  properties:
+                    message:
+                     type: "string"
+                     example: "Invalid event ID supplied, Event not found"
```

Comment:
This message does not match what you have written in your endpoint. It should be `Event not found` instead of `Invalid event ID supplied, Event not found`